src/helpers/tests/paths.test.ts

```diff
@@ -9,88 +9,100 @@ beforeEach(cleanup);
 afterAll(cleanup);
 
 describe("getFullModelsPaths", () => {
-  test("./dist/models", async () => {
+  test("./dist/models (ts)", async () => {
```

Comment:
Content of the test mentions "js" but title says "ts".
I'm confused, but I feel like this actually makes a lot of sense and it's just me being dumb at 4AM.

Reply:
No good catch, mixed up the title with the proceeding test!

Reply:
Fixed